sources/pmwebapi_test.go

```diff
@@ -54,7 +54,76 @@ func TestUnmarshal(t *testing.T) {
 		t.Fatalf("Not receiving correct URL. Expected: %s, Got: %s", expected, testMap.URL)
 	}
 }
+func TestWeirdMetrics(t *testing.T) {
```

Comment:
You could create slices or arrays with both the name and the expected value and iterate through using a loop. Would save code space and minimize redundancy at the same time.

Reply:
That sounds like a better idea, ill do that